CONTRIBUTING.md

```diff
@@ -11,21 +11,41 @@
 * `pnpm lint`
 * `pnpm lint:fix`
 
-## Building the addon
+## Developing
 
-* `cd ember-headless-form`
-* `pnpm build`
+To develop in this monorepo you can simply run:
 
-## Running tests
+ * `pnpm start`
```

Comment:
I am getting this error:

```
pnpm start

> ember-headless-form@0.0.0 start /Users/nchung/OpenSource/ember-headless-form
> concurrently 'npm:start:*(!only)' --names 'start:'

[start:tests]
[start:tests] > ember-headless-form@0.0.0 start:tests
[start:tests] > pnpm --filter test-app start
[start:tests]
[start:addon]
[start:addon] > ember-headless-form@0.0.0 start:addon
[start:addon] > pnpm --filter './packages/*' start --no-watch.clearScreen
[start:addon]
[start:docs]
[start:docs] > ember-headless-form@0.0.0 start:docs
[start:docs] > pnpm --filter docs-app start
[start:docs]
[start:tests]
[start:tests] > test-app@0.0.0 start /Users/nchung/OpenSource/ember-headless-form/test-app
[start:tests] > concurrently 'ember serve' 'sync-pnpm --watch' --names 'test-app serve,test-app sync built package'
[start:tests]
[start:addon] Scope: 3 of 7 workspace projects
[start:addon] packages/ember-headless-form start$ rollup --config --watch "--no-watch.clearScreen"
[start:docs]
[start:docs] > docs-app@0.0.1-beta.2 start /Users/nchung/OpenSource/ember-headless-form/docs-app
[start:docs] > concurrently 'ember serve' 'pnpm sync-pnpm --watch' --names 'docs-app serve,docs-app sync built package'
[start:docs]
[start:addon] packages/ember-headless-form start: (node:14852) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
[start:addon] packages/ember-headless-form start: (Use `node --trace-warnings ...` to show where the warning was created)
[start:addon] packages/ember-headless-form start: rollup v3.19.1
[start:addon] packages/ember-headless-form start: bundles  → dist...
[start:addon] packages/ember-headless-form start: (!) Generated empty chunks
[start:addon] packages/ember-headless-form start: "types" and "template-registry"
[start:addon] packages/ember-headless-form start: (!) Unused external imports
[start:addon] packages/ember-headless-form start: default imported from external module "@babel/runtime/helpers/esm/initializerWarningHelper" but never used in "src/components/headless-form.ts".
[start:addon] packages/ember-headless-form start: created dist in 2.1s
[start:tests] [test-app serve] Recrawled this watch 9 times, most recently because:
[start:tests] [test-app serve] MustScanSubDirs UserDroppedTo resolve, please review the information on
[start:tests] [test-app serve] https://facebook.github.io/watchman/docs/troubleshooting.html#recrawl
[start:tests] [test-app serve] To clear this warning, run:
[start:tests] [test-app serve] `watchman watch-del '/Users/nchung/OpenSource/ember-headless-form' ; watchman watch-project '/Users/nchung/OpenSource/ember-headless-form'`
[start:tests] [test-app serve]
[start:tests] [test-app serve] - building...
[start:docs] [docs-app serve] Building into /private/var/folders/jc/81dkjn9s2sg5cpt_66jd8l640000gn/T/embroider/c5718b
[start:docs] [docs-app serve] Recrawled this watch 9 times, most recently because:
[start:docs] [docs-app serve] MustScanSubDirs UserDroppedTo resolve, please review the information on
[start:docs] [docs-app serve] https://facebook.github.io/watchman/docs/troubleshooting.html#recrawl
[start:docs] [docs-app serve] To clear this warning, run:
[start:docs] [docs-app serve] `watchman watch-del '/Users/nchung/OpenSource/ember-headless-form' ; watchman watch-project '/Users/nchung/OpenSource/ember-headless-form'`
[start:docs] [docs-app serve]
[start:docs] [docs-app serve] - building...
[start:tests] [test-app serve] tests/unit/helpers/validate-changeset-test.ts:4:35 - error TS2307: Cannot find module 'ember-headless-form-changeset' or its corresponding type declarations.
[start:tests] [test-app serve]
[start:tests] [test-app serve] 4 import { validateChangeset } from 'ember-headless-form-changeset';
[start:tests] [test-app serve]                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[start:tests] [test-app serve]
[start:tests] [test-app serve] tests/unit/helpers/validate-yup-test.ts:3:29 - error TS2307: Cannot find module 'ember-headless-form-yup' or its corresponding type declarations.
[start:tests] [test-app serve]
[start:tests] [test-app serve] 3 import { validateYup } from 'ember-headless-form-yup';
[start:tests] [test-app serve]                               ~~~~~~~~~~~~~~~~~~~~~~~~~
[start:tests] [test-app serve]
[start:tests] [test-app serve] - building... [Babel: @ember/test-helpers > applyPatches]
[start:tests] [test-app serve] - building... [Babel: @ember/test-helpers > applyPatches]
[start:tests] [test-app serve] - building... [Babel: @ember/test-helpers > applyPatches]
[start:tests] [test-app serve] types/global.d.ts:5:39 - error TS2307: Cannot find module 'ember-headless-form/template-registry' or its corresponding type declarations.
[start:tests] [test-app serve]
[start:tests] [test-app serve] 5 import type HeadlessFormRegistry from 'ember-headless-form/template-registry';
[start:tests] [test-app serve]                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[start:tests] [test-app serve]
[start:tests] [test-app serve] Build Error (Funnel)
[start:tests] [test-app serve]
[start:tests] [test-app serve] ENOENT: no such file or directory, lstat '/Users/nchung/OpenSource/ember-headless-form/node_modules/.pnpm/file+packages+ember-headless-form_@glimmer+compone_du2l3pzj6lh4uzvqd2od2zolhu/node_modules/ember-headless-form/./dist/_app_/components/headless-form.js'
[start:tests] [test-app serve]
[start:tests] [test-app serve]
[start:tests] [test-app serve] Stack Trace and Error Report: /var/folders/jc/81dkjn9s2sg5cpt_66jd8l640000gn/T/error.dump.c2280a61eab3a331c98e2df6524645c9.log
[start:docs] [docs-app serve] Build Error (WaitForTrees)
[start:docs] [docs-app serve]
[start:docs] [docs-app serve] ember-headless-form/package.json lists ./dist/_app_/components/headless-form.js in ember-addon.app-js, but that file does not exist
[start:docs] [docs-app serve]
[start:docs] [docs-app serve]
[start:docs] [docs-app serve] Stack Trace and Error Report: /var/folders/jc/81dkjn9s2sg5cpt_66jd8l640000gn/T/error.dump.925d8daf00ffe21c310fc8407511e63a.log
^C[start:tests] [test-app serve] - cleaning up
[start:tests] [test-app serve] - cleaning up
[start:tests] [test-app serve] cleaning up...
[start:addon] packages/ember-headless-form start: Failed
[start:addon] undefined
[start:addon] /Users/nchung/OpenSource/ember-headless-form/packages/ember-headless-form:
[start:addon]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  ember-headless-form@1.0.0-beta.2 start: `rollup --config --watch "--no-watch.clearScreen"`
[start:addon] Command failed with signal "SIGINT"
[start:docs] [docs-app serve] - cleaning up
[start:docs] [docs-app serve] - cleaning up
[start:docs] [docs-app serve] cleaning up...
[start:addon] npm run start:addon exited with code 1
[start:docs] [docs-app sync built package] pnpm sync-pnpm --watch exited with code 1
[start:tests] [test-app sync built package] sync-pnpm --watch exited with code SIGINT
[start:tests] [test-app serve] ember serve exited with code 0
[start:tests] npm run start:tests exited with code 1
[start:docs] [docs-app serve] ember serve exited with code 0
[start:docs] npm run start:docs exited with code 1
```

To me it looks like it's starting the servers without building first.

What I have tried
1. clearing node_modules
2. `pnpm i`
3. `pnpm start`.

Reply:
Hmmm :'( Why `pnpm`, why??

I just did this:

```
git clone git@github.com:CrowdStrike/ember-headless-form.git testing123
cd testing123
git checkout watch-sync-pnpm
pnpm i
pnpm start
```

...and got the same :'( 

I'm not sure why it's working OK for me on my main checkout of the repo. My guess is it's a timing issue if you don't have the addon built at least once before you start - I'll try to dig in a bit more...

Reply:
Seems like a race condition where at least one build needs to have happened for things to work. Running `pnpm build` once makes everything work from then on. I'll figure out how to make that unnecessary on a clean checkout...